tests/output/ConsoleInteractionTest.py

```diff
@@ -712,9 +712,7 @@ def test_show_bear_with_content(self):
                              '  Needed Settings:\n'
                              '   * setting1: Required Setting.\n\n'
                              '  Optional Settings:\n'
-                             '   * setting2: Optional Setting. ('
-                             "Optional, defaults to 'None'."
-                             ')\n\n'
+                             '   * setting2: Optional Setting.\n\n'
```

Comment:
Hmm, I think we want this to be the same - the UI should have no real changes - the output for `--show-bears --show-description` should still have the `(Optional, defaults to 'whatever'.)` part. So store that string instead of just `Optional` (see the comment above), and append it just before printing. That way, we have access to both.

Reply:
@adtac Hi, I am the new assignee of the issue that this pull request is trying to solve. I read the conversation here and I am a bit confused about what the status of the issue is now. 

I understand that the goal is to keep the UI of the optional parameters output the same, as in a format like ```* tab_width: Number of spaces per indentation level. (Optional, defaults to '4'.)"```, and what we should do is somehow try to modify the ```((Optional, defaults to '4'.))``` which is concatenated to the string of the description of the parameter in FunctionMetadata.py. 

However, the part I don't understand is what is required for this modification. You mentioned that the reason you created the issue is that you wanted to get the default value for each setting to display as a column in bear-docs. But I thought the bear-docs is the cmd UI itself, and if we are displaying them in a column then the UI won't stay the same. Correct me if I am wrong for this part, but I am still trying to figure out what the difference that should be created is.

Also, if we discard the ```((Optional, defaults to '4'.))``` which is concatenated to the description of the parameter, then we need to modify the ```show_enumeration``` function in ```ConsoleInteraction.py``` just like @dracarys09 said. I think this is doable but we need to create a special case just to deal with ```optional parameters``` in ```show_enumeration```. I just want to make sure I understand the requirement before I make any attempt at it.